### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by schriftgestalt**

### CODE REVIEW

File: Lib/vanilla/test/testAll.py [hunk 1]
```diff
@@ -15,8 +15,8 @@
 
 import objc
 objc.setVerbose(True)
-
-vanillaPath = os.path.dirname(os.path.dirname(os.path.dirname(vanilla.__file__)))
+vanillaPath = NSString.stringWithString_(vanilla.__file__).stringByResolvingSymlinksInPath()
```
Comment (justvanrossum) [inline Lib/vanilla/test/testAll.py]: Wouldn't `vanillaPath = os.path.realpath(vanilla.__file__)` do the same thing?

File: Lib/vanilla/vanillaLevelIndicator.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 # This control is available in OS 10.4+.
 # Cause a NameError if in an earlier OS.
-NSLevelIndicator
+# NSLevelIndicator
```
Comment (justvanrossum) [inline Lib/vanilla/vanillaLevelIndicator.py]: Read the comment above this line :)

Comment (justvanrossum) [reply Lib/vanilla/vanillaLevelIndicator.py]: On the other hand, I doubt we meaningfully support 10.4, so maybe that entire section should go.

File: Lib/vanilla/test/testAll.py (hunk 1 above)
Comment (schriftgestalt) [reply Lib/vanilla/test/testAll.py]: I tried to find how that would be done in python and google didn't tell me. 
Your code works fine. I’ll change it.

File: Lib/vanilla/vanillaLevelIndicator.py (hunk 2 above)
Comment (schriftgestalt) [reply Lib/vanilla/vanillaLevelIndicator.py]: Now I see.

**Pull request merged**